features/step_definitions/work_search_steps.rb

```diff
@@ -134,16 +134,21 @@
 Given /^a set of works with comments for searching$/ do
   step %{basic tags}
 
-  # Comments created with factories are not added to a work's stat totals
-  # even after running the rake task, so we're doing it through steps that add
-  # comments through the interface
-  step %{I have a work "Work 1"}
-  step %{the work "Work 2" with 1 comments setup}
```

Comment:
Do you think we should update this step as well? Using factories will probably be faster looping through the UI:

https://github.com/otwcode/otwarchive/blob/37fa9b799f39d76fe3cde07d920ddf3abf451af5/features/step_definitions/work_steps.rb#L163-L173

Reply:
Sure, that seems reasonable. Do you want me to create the work using `FactoryBot`, too? Or just the comments?

And should I do it for this step, too? https://github.com/otwcode/otwarchive/blob/a7bb25ac11754fd8079b4ac9d6c877f541735842/features/step_definitions/work_steps.rb#L125-L146

Reply:
Please and thanks! Factories for work + comments, in both steps.